R/create.R

```diff
@@ -32,14 +40,14 @@ quarto_create_project <- function(name, type = "default", dir = ".", no_prompt =
       "This will create a new Quarto {.emph {type}} project as a folder named {.strong {name}} in {.path {xfun::normalize_path(dir)}}."
     ))
     prompt_value <- tolower(readline(sprintf("Do you want to proceed (Y/n)? ")))
-    if (!prompt_value %in% "y") {
+    if (!prompt_value %in% c("", "y")) {
       cli::cli_abort("Operation aborted.")
     }
   }
 
   quarto_bin <- find_quarto()
 
-  args <- c("project", type, name, "--no-prompt", "--no-open", if (is_quiet(quiet)) cli_arg_quiet(), quarto_args = NULL)
+  args <- c("project", type, name, name, "--no-prompt", "--no-open", if (is_quiet(quiet)) cli_arg_quiet(), quarto_args = NULL)
```

Comment:
Without this, I always get a warning, coming from the quarto cli, with no way to suppress it.

```
> quarto_create_project("my-first-quarto-project", dir = "~/tmp")
This will create a new Quarto default project as a folder named my-first-quarto-project in [/Users/jenny/tmp](vscode-file://vscode-app/Applications/Positron.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html#).
Do you want to proceed (Y/n)? 
WARN: No 'title' for project provided in `quarto create project`. Using 'my-first-quarto-project' as default.
Creating project at /Users/jenny/tmp/my-first-quarto-project:
  - Created _quarto.yml
  - Created my-first-quarto-project.qmd
  - ```